cobbler/actions/reposync.py

```diff
@@ -348,13 +348,39 @@ def rsync_sync(self, repo):
         if not repo.mirror.strip().endswith("/"):
             repo.mirror = "%s/" % repo.mirror
 
-        # FIXME: wrapper for subprocess that logs to logger
-        cmd = "rsync -rltDv --copy-unsafe-links --delete-after %s --delete --exclude-from=/etc/cobbler/rsync.exclude %s %s" % (spacer, pipes.quote(repo.mirror), pipes.quote(dest_path))
+        flags = ''
+        for x in repo.rsyncopts:
+            if repo.rsyncopts[x]:
+                flags += " %s %s" % (x, repo.rsyncopts[x])
+            else:
+                flags += " %s" % x
+        
+        if flags == '':
+            flags = self.settings.reposync_rsync_flags
+
+        cmd = "rsync %s --delete-after %s --delete --exclude-from=/etc/cobbler/rsync.exclude %s %s" % (flags, spacer, pipes.quote(repo.mirror), pipes.quote(dest_path))
         rc = utils.subprocess_call(self.logger, cmd)
 
         if rc != 0:
             utils.die(self.logger, "cobbler reposync failed")
-        repo_walker(dest_path, self.createrepo_walker, repo)
+        
+        # if ran in archive mode then repo should already contain all repodata and does not need createrepo ran 
```

Comment:
should be "does not need createrepo run" instead of "createrepo ran"...

Reply:
updated comment